### PR TITLE
edencommon fb303 fbthrift fizz folly wangle watchman 2023.02.20.00

### PR DESCRIPTION
**Pull request by ZhongRuoyu**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by chenrui333** [comment]
```
  error[E0583]: file not found for module `facebook`
    --> src/rage/mod.rs:33:1
     |
  33 | mod facebook;
     | ^^^^^^^^^^^^^
     |
     = help: to create the module `facebook`, create file "src/rage/facebook.rs" or "src/rage/facebook/mod.rs"
  
  error[E0432]: unresolved import `super::facebook::FbReporter`
    --> src/rage/stream.rs:13:9
     |
  13 | pub use super::facebook::FbReporter;
     |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^ no `FbReporter` in `rage::facebook`
  
  Some errors have detailed explanations: E0432, E0583.
```

**Comment by carlocab** [comment]
Try passing `-DIS_FB_BUILD=NO` to `cmake`.

**Comment by carlocab** [comment]
Nope, doesn't work. Needs an upstream report.

**Comment by carlocab** [comment]
Here's the [problem](https://github.com/facebook/watchman/blob/7f619ba5f170423744cb7778dccb0bb160f0a0d8/watchman/cli/Cargo.toml#L32):

```toml
default = ["fb"]
```
Needs to say
```toml
default = []
```

**Comment by ZhongRuoyu** [comment]
Filed https://github.com/facebook/watchman/issues/1099.

File: Formula/watchman.rb [hunk 1]
```diff
@@ -44,6 +44,10 @@ def install
               /gtest_discover_tests\((.*)\)/,
               "gtest_discover_tests(\\1 DISCOVERY_TIMEOUT 30)"
 
+    # Workaround for build failure. Reported at:
+    # https://github.com/facebook/watchman/issues/1099
+    inreplace "watchman/cli/Cargo.toml", 'default = ["fb"]', "default = []"
```
Comment (carlocab) [inline Formula/watchman.rb:49]: Looks like it worked.

**Review by carlocab — APPROVED** [review]
Big Sur was cancelled while testing dependents (but there are no dependents to test). This should be fine to merge after I check the bottles.

**Comment by ZhongRuoyu** [comment]
> Big Sur was cancelled while testing dependents (but there are no dependents to test).

I can recall this happening every time. :/

**Comment by carlocab** [comment]
49 bottles -- all there.

**Pull request closed without merging**